ext/pdo/pdo_dbh.stub.php

```diff
@@ -394,6 +394,9 @@ public function beginTransaction(): bool {}
     /** @tentative-return-type */
     public function commit(): bool {}
 
+    /** @tentative-return-type */
```

Comment:
you don't need to add tentative return type for new methods. It's purely for BC of existing methods.